### PR TITLE
clarify sdt doc

### PR DESCRIPTION
**Pull request by haimeiguo**
variable name clarify
1. compMax -> compMaxTime
2. add default value and unit

### CODE REVIEW

File: docs/zh/UserGuide/Concept/SDT.md [hunk 1]
```diff
@@ -29,15 +29,15 @@ IoTDB允许您在创建时间序列时指定SDT的属性，并支持以下三个
...
+CompDev是SDT中最重要的参数，代表当前样本和当前线性之间的最大差趋势。默认为0。
 
-* CompMin (最小压缩间隔)
+* CompMinTime(最小压缩时间间隔)
 
-CompMin主要用于减少噪点。 CompMin测量两个存储的数据点之间的时间距离，如果当前点的时间到上一个存储的点的时间距离小于或等于compMin， 无论压缩偏差值，都不会存储当前数据点。
+CompMinTime主要用于减少噪点。 CompMinTime测量两个存储的数据点之间的时间距离，如果当前点的时间到上一个存储的点的时间距离小于或等于CompMinTime， 无论压缩偏差值，都不会存储当前数据点。默认为0，时间单位为ms。
 
-* CompMax (最大压缩间隔)
+* CompMaxTime (最大压缩时间间隔)
```
Comment (SteveYurongSu) [inline docs/zh/UserGuide/Concept/SDT.md:38]:
```suggestion
* CompMaxTime （Compression Maximum Time Interval，最大压缩时间间隔）
```

File: docs/UserGuide/Concept/SDT.md [hunk 2]
```diff
@@ -30,19 +30,21 @@ IoTDB allows you to specify the properties of SDT when creating a time series, a
 
 * CompDev (Compression Deviation)
 
-CompDev is the most important parameter in SDT that represents the maximum difference between the current sample and the current linear trend.
+CompDev is the most important parameter in SDT that represents the maximum difference between the 
+current sample and the current linear trend. Default value is 0.
```
[the remark this answers is not in the document]
Comment (haimeiguo) [reply docs/UserGuide/Concept/SDT.md:34]: if use SDT and not set compDev or compDev is set to < 0, will throw error..

File: docs/UserGuide/Concept/SDT.md [hunk 3]
```diff
@@ -30,19 +30,21 @@ IoTDB allows you to specify the properties of SDT when creating a time series, a
...
 
-* CompMin (Compression Minimum)
+* CompMinTime (Compression Minimum Time Interval)
 
-CompMin is used for noisy reduction. CompMin measures the time distance between two stored data points, 
-if the current point's time to the last stored point's time distance is smaller than or equal to compMin, 
-current point will NOT be stored regardless of compression deviation.
+CompMinTime is used for noisy reduction. CompMinTime measures the time distance between two stored data points, 
+if the current point's time to the last stored point's time distance is smaller than or equal to CompMinTime, 
+current point will NOT be stored regardless of compression deviation. Default value is 0 and time unit is ms.
```
Comment (SteveYurongSu) [inline docs/UserGuide/Concept/SDT.md:40]: What if we don't set CompMinTime and CompMaxTime ?

File: docs/UserGuide/Concept/SDT.md [hunk 4]
```diff
@@ -30,19 +30,21 @@ IoTDB allows you to specify the properties of SDT when creating a time series, a
 
 * CompDev (Compression Deviation)
 
-CompDev is the most important parameter in SDT that represents the maximum difference between the current sample and the current linear trend.
+CompDev is the most important parameter in SDT that represents the maximum difference between the 
+current sample and the current linear trend. CompDev needs to be greater than 0 to perform compression.
 
-* CompMin (Compression Minimum)
+* CompMinTime (Compression Minimum Time Interval)
```
Comment (SteveYurongSu) [inline docs/UserGuide/Concept/SDT.md:36]: The code should also be modified according to the documentation.

**Review by SteveYurongSu — CHANGES_REQUESTED** [review]

File: docs/UserGuide/Concept/SDT.md [hunk 5]
```diff
@@ -30,19 +30,21 @@ IoTDB allows you to specify the properties of SDT when creating a time series, a
...
 
-CompMin is used for noisy reduction. CompMin measures the time distance between two stored data points, 
-if the current point's time to the last stored point's time distance is smaller than or equal to compMin, 
-current point will NOT be stored regardless of compression deviation.
+CompMinTime is used for noisy reduction. CompMinTime measures the time distance between two stored data points, 
+if the current point's time to the last stored point's time distance is smaller than or equal to CompMinTime, 
+current point will NOT be stored regardless of compression deviation. Default value is -2147483647 and time unit is ms.
```
Comment (HTHou) [inline docs/UserGuide/Concept/SDT.md:40]:
```suggestion
if the time interval between the current point and the last stored point is less than or equal to this value, current point will NOT be stored regardless of compression deviation. 
The Default value is -2147483647 with time unit ms.
```

Comment (HTHou) [inline docs/UserGuide/Concept/SDT.md:40]:
```suggestion
CompMinTime is a parameter measures the time distance between two stored data points, which is used for noisy reduction.
If the time interval between the current point and the last stored point is less than or equal to its value, current point will NOT be stored regardless of compression deviation. 
The Default value is -2147483647 with time unit ms.
```

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java [hunk 6]
```diff
@@ -139,21 +139,21 @@ private void checkSdtEncoding() {
 
       if (isSdtEncoding && measurementSchema.getProps().containsKey("compmin")) {
```
Comment (SteveYurongSu) [inline tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java:140]: Don't forget to modify the most important code here. Please have a manual test.

`compmin` -> `compmintime`

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java [hunk 7]
```diff
@@ -139,21 +139,21 @@ private void checkSdtEncoding() {
...
         try {
-          sdtEncoder.setCompMin(Double.parseDouble(measurementSchema.getProps().get("compmin")));
+          sdtEncoder.setCompMinTime(Double.parseDouble(measurementSchema.getProps().get("compmin")));
```
Comment (SteveYurongSu) [inline tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java:142]: `compmin` -> `compmintime`

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java [hunk 8]
```diff
@@ -139,21 +139,21 @@ private void checkSdtEncoding() {
...
         } catch (NumberFormatException e) {
           logger.error("meet error when formatting SDT compression minimum");
         }
       }
 
       if (isSdtEncoding && measurementSchema.getProps().containsKey("compmax")) {
```
Comment (SteveYurongSu) [inline tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java:148]: `compmax` -> `compmaxtime`

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java [hunk 9]
```diff
@@ -139,21 +139,21 @@ private void checkSdtEncoding() {
...
         try {
-          sdtEncoder.setCompMax(Double.parseDouble(measurementSchema.getProps().get("compmax")));
+          sdtEncoder.setCompMaxTime(Double.parseDouble(measurementSchema.getProps().get("compmax")));
```
Comment (SteveYurongSu) [inline tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java:150]: `compmax` -> `compmaxtime`

**Review by SteveYurongSu — CHANGES_REQUESTED** [review]

File: docs/zh/UserGuide/Concept/SDT.md [hunk 10]
```diff
@@ -27,17 +27,22 @@
 
 IoTDB允许您在创建时间序列时指定SDT的属性，并支持以下三个属性：
 
-* CompDev (压缩偏差)
+* CompDev （Compression Deviation，压缩偏差）
 
-CompDev是SDT中最重要的参数，代表当前样本和当前线性之间的最大差趋势。
+CompDev是SDT中最重要的参数，代表当前样本和当前线性之间的最大差趋势。CompDev设置的值需要大于0。
```
Comment (SteveYurongSu) [inline docs/zh/UserGuide/Concept/SDT.md:32]:
```suggestion
CompDev是SDT中最重要的参数，它表示当前样本与当前线性趋势之间的最大差值。CompDev设置的值需要大于0。
```

File: docs/zh/UserGuide/Concept/SDT.md [hunk 11]
```diff
@@ -27,17 +27,22 @@
...
 
-* CompMin (最小压缩间隔)
+* CompMinTime （Compression Minimum Time Interval，最小压缩时间间隔）
 
-CompMin主要用于减少噪点。 CompMin测量两个存储的数据点之间的时间距离，如果当前点的时间到上一个存储的点的时间距离小于或等于compMin， 无论压缩偏差值，都不会存储当前数据点。
+CompMinTime是测量两个存储的数据点之间的时间距离的参数，用于减少噪声。
+如果当前点和最后存储的点之间的时间间隔小于或等于其值，则无论压缩偏差如何，都不会存储当前点。
+默认值为-2147483647，单位为毫秒。
```
Comment (SteveYurongSu) [inline docs/zh/UserGuide/Concept/SDT.md:38]: Setting the default value to 0 is better.

File: docs/zh/UserGuide/Concept/SDT.md [hunk 12]
```diff
@@ -27,17 +27,22 @@
...
 
-* CompMax (最大压缩间隔)
+* CompMaxTime （Compression Maximum Time Interval，最大压缩时间间隔）
 
-CompMax用于定期检查上一个存储的点到当前点之间的时间距离。它测量存储点之间的时间差。如果当前点时间到上一个存储点的时间距离 大于或等于compMax，无论压缩偏差值，都会存储当前数据点。
+CompMaxTime是测量两个存储的数据点之间的时间距离的参数。
+如果当前点和最后一个存储点之间的时间间隔大于或等于其值，
+无论压缩偏差如何，都将存储当前点。
+默认值为2147483647，单位为毫秒。
```
Comment (SteveYurongSu) [inline docs/zh/UserGuide/Concept/SDT.md:45]: Setting the default value to Long.MAX_VALUE is better.

**Review by SteveYurongSu — APPROVED** [review]
LGTM. Thanks for your contribution!

**Pull request merged**